changelogs/fragments/6990.yml

```diff
@@ -0,0 +1,2 @@
+fix:
+- Hide duplicated scrollbars. ([#5616](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5616))
```

Comment:
nit: think this needs to be indented to fix the workflow failure.

Reply:
+1

Reply:
The issue is in table vis. We use embeddable to add table vis in Dashboard. Therefore, update table vis should limit the impact to local. Table vis is in src/plugins/vis_type_table.